release/support-rulesengine.yaml

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
I dont think we are release support-rulesengine anymore. The repo has been archived

https://github.com/edgexfoundry/support-rulesengine.git
> This repository has been archived by the owner. It is now read-only.

Reply:
@jpwhitemn , @lenny-intel  can you confirm?  I think per our discussion we still need `docker-edgex-mongo`, `docker-support-logging-go` and `support support-rulesengine` for Hanoi correct?

Reply:
> we still need `docker-edgex-mongo`, `docker-support-logging-go` and `support support-rulesengine` for Hanoi correct?

@bill-mahoney , that is correct.

Reply:
We haven't made any changes to support-rules engine, so I assume we could just use the Geneva released image? @jpwhitemn , Thoughts?

Reply:
Not that we are going to actual `use` it anywhere, but it is still available.

Reply:
talked in slack, removed it from PR for Hanoi